bundles/org.openhab.binding.echonetlite/src/main/test/org/openhab/binding/echonetlite/internal/EpcTest.java

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
In order to be discovered and run by Maven, this should be moved from:
- bundles/org.openhab.binding.echonetlite/src/main/test/org/openhab/binding/echonetlite/internal/ to:
- bundles/org.openhab.binding.echonetlite/src/test/java/org/openhab/binding/echonetlite/internal/

Reply:
That was weird not sure how the tests ended up where they did.